util/countries/australia/territories.json

```diff
@@ -6,5 +6,6 @@
 	{"short": "SA", "name": "South Australia", "country": "AU"},
 	{"short": "TAS", "name": "Tasmania", "country": "AU"},
 	{"short": "VIC", "name": "Victoria", "country": "AU"},
-	{"short": "WA", "name": "Western Australia", "country": "AU"}
+	{"short": "WA", "name": "Western Australia", "country": "AU"},
+	{"short": "NULL", "name": "No State / Province", "country": "AU"}
```

Comment:
I don't think adding a NO STATE option to each countries province/state list is the best solution. It would be hard to scale in the future.